peps/pep-0788.rst

```diff
@@ -684,35 +525,28 @@ With this PEP, you'd implement it like this:
         free(to_write);
         PyErr_Print();
```

Comment:
I'm not sure that that do_some_text_mutation() is useful. Maybe just remove it.

What is the PyErr_Print() call? Is it useful? Maybe also call it if res is negative?